binding/web/src/cheetah.ts

```diff
@@ -89,15 +89,17 @@ export class Cheetah {
 
   private static _cheetahMutex = new Mutex();
 
+  private _isDetached: boolean = false;
```

Comment:
just so we're clear what this means, it should be called `_isWasmMemoryDetached` or something